jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -587,6 +589,12 @@ function configure_jbpm() {
     fi
 }
 
+function configure_optaplanner() {
+    if [ "${JBOSS_PRODUCT}" = "rhpam-kieserver" ]; then
```

Comment:
Optaplanner can be in either rhdm-kieserver or rhpam-kieserver, so this if statement is wrong. Instead, the if statement should check that the variable is actually set. Thus, I think this line should be:

if [ -n "${OPTAPLANNER_SERVER_EXT_THREAD_POOL_QUEUE_SIZE}" ]; then

Reply:
Done!